3DAmsterdam/Assets/Amsterdam3D/Scripts/BAG-API/PandButton.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Events;
+
+public class PandButton : MonoBehaviour
+{
+    public Button btn;
```

Comment:
liever geen afkortingen gebruiken ( mag niet volgens code conventie )
in dit geval is het overduidelijk dat het om een button gaat, maar voor de consistentie liever volledig uitgeschreven.

Reply:
Is opgelost